src/constants/autocomplete-options.js

```diff
@@ -1,45 +1,20 @@
-function sleep(ms) {
-  return new Promise(resolve => setTimeout(resolve, ms));
-}
+import { get } from '@/utils/api.js'; 
```

Comment:
Okay, and this file was supposed to remain constant :) It feels like we have a certain level of misunderstanding as to what should go in `utils/` and what should go to `constants/`. I'd rather revert this file for the sake of the UI page which uses it in constant format and move this functionality to `utils/`. Anyway, we should probably talk this over in person